python/hail/docs/scans.rst

```diff
@@ -0,0 +1,78 @@
+.. _sec-scan:
+
+Scans
+===========
+
+The ``scan`` module is exposed as ``hl.scan``, e.g. ``hl.scan.sum``.
+
+The functions in this module perform rolling aggregations along the desired axis;
```

Comment:
The functions in this module perform rolling aggregations along the rows of a table, or along the rows or columns of a matrix table. The value of the scan at a given row (resp., column) is the result of applying the corresponding aggregator to all previous rows (resp., columns). Scans directly over entries are not currently supported.